Simplenote/MagicLinkAuthenticator.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
```

Comment:
Oh!! I've just noticed this file isn't in the `Classes/` folder, would it be possible to move it?

Reply:
Ummmm I'm thinking we may need to go even further. Because... well, this is not a class, isn't it!😂

Reply:
Argh, I happens all the time to me :-(